pkgs/by-name/x2/x2t/package.nix

```diff
@@ -0,0 +1,617 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  boost,
+  icu,
+  freetype,
+  qt5,
+  harfbuzz,
+  # needs to be static and built with MD2 support!
+  openssl,
+  runCommand,
+  nodejs,
+  onlyoffice-documentserver,
+  writeScript,
+  x2t,
+}:
+
+let
+  qmake = qt5.qmake;
+  libv8 = nodejs.libv8;
+  fixIcu = writeScript "fix-icu.sh" ''
+    substituteInPlace \
+      /build/source/Common/3dParty/icu/icu.pri \
```

Comment:
`/build` is an implementation detail that isn't consistent (e.g. won't exist if building unsandboxed) and shouldn't be referenced anywhere. You probably want just `Common/3dParty/icu/icu.pri` (and same thing for all the other occurrences of `/build`

Reply:
good point, thanks! I'll replace it with `$NIX_BUILD_TOP` so it'll work independently of what `sourceRoot` is set to, which many of these components use.

Reply:
Using NIX_BUILD_TOP isn't a good idea: https://github.com/nixos/nixpkgs/issues/189691